Docs: RBAC GA (#49062)
grafana/grafana: docs/sources/administration/manage-users-and-permissions/manage-dashboard-permissions/_index.md

This modification is useful for public Grafana installations where you want anonymous users to be able to edit panels and queries but not save or create new dashboards.

> > **Note**: If you are using Grafana Enterprise and customize users permissions using RBAC, the RBAC permissions will override the functionality enabled by `viewers_can_edit` flag.

Reviewer: ```suggestion
> **Note**: If you use Grafana Enterprise and customize users' permissions using RBAC, the RBAC permissions override the functionality enabled by the `viewers_can_edit` flag.
```
